docs/source/user_guide.rst

```diff
@@ -53,6 +53,45 @@ calling ``result()`` on ``fut`` above forces the program to wait for the
 job started with :func:`~civis.io.dataframe_to_civis` to finish and
 returns the result.
 
+You can create :class:`CivisFuture <civis.futures.CivisFuture>` objects for
+many tasks (e.g., scripts, imports):
```

Comment:
This is great! If you don't mind, it would be nice to have 2 or so sentences here just describing high level what the code below is doing.  Perhaps something like

> Here we will be creating a container script that does the simple task of printing the text "HELLO WORLD".  We will be creating the script, executing it and then waiting for the script to finish.